src/components/containers/seo.js

```diff
@@ -4,12 +4,14 @@ import { Helmet } from 'react-helmet'
 import { useStaticQuery, graphql } from 'gatsby'
 import { LocaleContext } from '../localization'
 import language_config from '../../../i18n-config'
-import TradingImage from 'images/common/practice.png'
+import TradingImage from 'images/common/og_deriv.png'
 
 const non_localized_links = ['/careers', '/careers/']
 
 const languages = Object.keys(language_config)
 languages.push('x-default')
+const default_og_title = 'Online trading with Deriv | Simple. Flexible. Reliable.'
+const default_og_description = 'Trading platforms designed with you in mind.'
```

Comment:
You'll need to localize these strings as well. Could we also keep these variables nested within the SEO component? 🙏

Reply:
Noted, @sean-binary